### PR TITLE
Allow changing person info based on user role

### LINKED ISSUE #241: Adjust Register User button format based on user role
@tristanlatr  @victorphoenix3  I'm really not a big fan of the big `Register User` button .I am okay to keep a large button for newly added  **core members** and **pickleaders** but here it's an invitation to set a password for owners and volunteer pickers, which is not what we want to do at the moment (for this season at least). 
So I'd suggest either we go back to something more subtle or we at least check for the user role before presenting a large button.

screenshot_2022-04-14_10:08:27

### PR DESCRIPTION
**Pull request by victorphoenix3**
Fixes #241

### CODE REVIEW

**Review by 2ynn — APPROVED** [review]
Thank you so much!

File: saskatoon/sitebase/templates/app/list_views/community/data_row.html [hunk 1]
```diff
@@ -24,15 +24,17 @@
...
-            <a href="/admin/member/authuser/{{user.id}}/password/"
-               onclick="return confirm('{% trans "Are you sure you want to reset this user password?" %}');"
-            {% if not user.password %}
-                title="register-user"
-                >
-                <small class="btn btn-primary  waves-effect btn-sm"><i class="fa fa-plus"> </i> {% translate ' Register User' %} </a></small>
-            {% else %}
-                title="reset-password">
-                <small><i class="fa fa-refresh"></i></a></small>
+            {% if 'core' in user.role_codes or 'pickleader' in user.role_codes %}
```
Comment (tristanlatr) [inline saskatoon/sitebase/templates/app/list_views/community/data_row.html:27]: I think only core members or admins should be able to reset one's password. Not pickleaders.

Comment (tristanlatr) [reply saskatoon/sitebase/templates/app/list_views/community/data_row.html:27]: Ok sorry for my confusion.

**Comment by tristanlatr** [comment]
I’ll let you merge this PR @2ynn, thanks a lot @victorphoenix3 :)

**Pull request merged**